WordPress/Classes/Extensions/UITableView+Helpers.swift

```diff
@@ -31,4 +39,17 @@ extension UITableView
             self.deselectSelectedRowWithAnimation(animated)
         }
     }
+
+    /// Disables Editing after a specified delay.
+    ///
+    /// -   Parameter delay: milliseconds to elapse before edition will be disabled.
+    ///
+    public func disableEditionAfterDelay(delay: NSTimeInterval = 0.2) {
```

Comment:
It'd be the 'Default Parameter Value'. Swift trick to get away to... not to send a parameter value, and still, get a default behavior.

Thought it wouldn't pick up a constant, but it worked! updating right away, thanks!